File: 02_algorithms/01_prime_number/prime_number.py
def prime_number(number: int) -> None:
    if number <= 1:
        return

    for i in range(2, number, 1):
        if number % i == 0:
            return

    print(f'{number} is Prime')


if __name__ == '__main__':
    for x in range(0, 11, 1):
        prime_number(x)

File: 02_algorithms/01_prime_number/prime_number.c
#include <stdio.h>
#include <stdlib.h>

void prime_number(int number) {
  if (number <= 1) {
    return;
  }

  for (int i = 2; i < number; i++) {

    if (number % i == 0) {
      return;
    }
  }

  printf("\n%d is Prime", number);

  return;
}

int main() {    
  for (int x = 0; x <= 11; x++) {
    prime_number(x);
  }
}